administrator/components/com_media/src/Controller/AdaptiveImageController.php

```diff
@@ -64,7 +64,8 @@ public function execute($task)
 				break;
 			case "cropImage" :
 				$imgPath = $this->input->getString('path');
-				$this->cropImage($imgPath);
+				$widths = json_decode($this->input->getString('widths'), true);
+				$this->cropImage($imgPath, $widths);
 				return true;
 				break;
```

Comment:
i think we can safely remove break as it will be never executed

Reply:
I think ```return``` statement should be removed.

Reply:
Leave the returns, remove the breaks.